imagine/imagine.py

```diff
@@ -17,11 +18,21 @@
 import click
 from PIL import Image
 from multiprocessing.pool import Pool
-from mxnet.recordio import IRHeader, MXIndexedRecordIO, pack
+try:
+    from mxnet.recordio import IRHeader, MXIndexedRecordIO, pack
+except ImportError:
+    IRHeader = None
```

Comment:
This was my main question - how do we message that further dependencies are necessary? Looks alright to me.

Reply:
Sadly, I don't think there's a great, clean way to really do this, but the way I implemented it seems to be the most commonly preferred method.

Reply:
Ok, just so long as you did some investigation into what options exist, I'm fine with this.